website/pages/api-docs/acl/auth-methods.mdx

```diff
@@ -70,6 +70,31 @@ The table below shows this endpoint's support for
   If not provided at all, the namespace will be inherited from the request's ACL
   token or will default to the `default` namespace. Added in Consul 1.7.0.
 
+- `NamespaceRules` `(array<NamespaceRule>)` <EnterpriseAlert inline /> - A set
+  of rules that can control which namespace tokens created via this auth method
+  will be created within. Note that assigning namespaces via rules requires the
```

Comment:
Should we sub "will be created within" to something like "will be associated with"?

Reply:
Tokens are stored within a namespace. If you delete a namespace everything inside it is erased.

Reply:
Okay, I see. Intentional wording is worded well 👍